style.css

```diff
@@ -0,0 +1,235 @@
+* {
```

Comment:
completely valid CSS but we don't recommend directly targeting HTML tags because as applications get larger it can create bugs and unwanted "side effects"